pkg/audit/manager.go

```diff
@@ -192,7 +193,7 @@ func getUpdateListsFromAuditResponses(resp *constraintTypes.Responses) (map[stri
 			rname:             rname,
 			rnamespace:        rnamespace,
 			message:           message,
-			enforcementaction: "deny", // default value to "deny" until we have more actions to support
```

Comment:
I forgot that we had already hardcoded deny in here. Technically it's a breaking change to change the case. We should get a feel for if people prefer all-upper or all lower, or don't care.

If (2) or (3) we should keep with lower to keep the API consistent.

Reply:
I think this hardcoded deny is only in the `violations` field of `status` as part of the audit results of a constraint. So if users run a new release of gatekeeper, this value should be updated after the first audit run and it should not have any impact on validation. I will bring this up to the group in today's call to see if anyone has a strong preference to any of these options.

Reply:
Discussed this on our weekly call. The team prefers to use lower case `deny` instead. Will update the PR on /frameworks to reflect this decision.

cc @teq0 @timothyhinrichs